### ISSUE TITLE
Drop-down menu disappears when using OpenGL shaders on Linux Mint 21

### ISSUE THREAD

**Issue by kneelian** [joncampbell123/dosbox-x#4343]
### Describe the bug

Selecting an OpenGL shader from [this repository](https://github.com/tyrells/dosbox-svn-shaders) causes drop-down menu to disappear. Using the F12-Esc shortcut does not re-enable menu. Shortcut toggles menu as expected when no shader loaded.

### Steps to reproduce the behaviour

1. Install recent build of DOSBox-X
2. Download e.g. CRT shaders from aforementioned repository
3. Select a shader via dropdown menu or config menu --> Render

### Expected behavior

No disappearance of drop-down menu, ability to toggle menu.

### What operating system(s) this bug have occurred on?

Linux Mint 21.1 (kernel v. 5.15.0-56-generic)

### What version(s) of DOSBox-X have this bug?

2023.05.01 (64-bit SDL2)

### Output log

```text
LOG: Aspect ratio: 720 x 540  xToY=1.333 yToX=0.750
LOG: menuScale=2
LOG: OpenGL PP: [1920x1012]: 720x400 (1.00) -> [2x2] -> 1440x800 (1.00)
LOG: Screen report: Method 'None' (-1.000 x -1.000 pixels) at (0.000 x 0.000) (-1.000 x -1.000 mm) (-0.039 x -0.039 in) (-1.000 x -1.000 DPI)
LOG: font texture id=2 will make 128 x 256
LOG: Screen report: Method 'None' (-1.000 x -1.000 pixels) at (0.000 x 0.000) (-1.000 x -1.000 mm) (-0.039 x -0.039 in) (-1.000 x -1.000 DPI)
```

### Have you checked that no similar bug report(s) exist?

- [X] I have searched and didn't find any similar bug report.

### Code of Conduct & Contributing Guidelines

- [X] I agree to follow the code of conduct and the contributing guidelines.

**Comment by GuybrushSF** [comment]
Same problem with Dosbox-x 2023-09-01 and Mint 21-2

Without GLShader : 
![image](https://github.com/joncampbell123/dosbox-x/assets/81114951/2ee5fb93-06bc-4e0d-9b5d-70c09246b581)
With GLShader crt-lottes-fast.subtle+gain.glsl
![image](https://github.com/joncampbell123/dosbox-x/assets/81114951/910f83eb-2a51-4e30-964d-b6c7a7733edf)

**Comment by rderooy** [comment]
Yes this is a known limitation, and duplicate issue. see also https://github.com/joncampbell123/dosbox-x/issues/2144

**Comment by joncampbell123** [comment]
OpenGL has to share state between DOSBox-X and the guest and it's rather hard in that case to also do the menu bar without host OS support.
